DICTIONARY.md

```diff
@@ -40,7 +40,7 @@
 * **default** — за замовчуванням
 * **deferreds** — де́фереди
 * **derived objects** — похідні об’єкти
-* **Destructuring assigment** — Деструктивне присвоєння
+* **Destructuring assigment** — Деструктуруюче присвоєння
```

Comment:
Please, move this conversation to the [dictionary](https://github.com/LambdaBooks/dictionary).